Update tutorial.rst

Mubarraqqq opened the pull request:
Hi @joshmoore @MSanKeys963. 
I made an update on the Zarr Tutorial and also included a new compressor.  

TODO:
* [ ] Add unit tests and/or doctests in docstrings
* [ ] Add docstrings and API docs for any new/modified user-facing classes and functions
* [ ] New/modified features documented in docs/tutorial.rst
* [ ] Changes documented in docs/release.rst
* [ ] GitHub Actions have all passed
* [ ] Test coverage is 100% (Codecov passes)

MSanKeys963 commented on docs/tutorial.rst:
```diff
@@ -204,6 +204,47 @@ here is an array using Zstandard compression, level 1::
 
+Getting the diagnostics on the compression ratio :
+
+    >>> z.info
+    Type               : zarr.core.Array
+    Data type          : int32
+    Shape              : (10000, 10000)
+    Chunk shape        : (1000, 1000)
+    Order              : C
+    Read-only          : False
+    Compressor         : Zstd(level=1)
+    Store type         : zarr.storage.KVStore
+    No. bytes          : 400000000 (381.5M)
+    No. bytes stored   : 299348530 (285.5M)
+    Storage ratio      : 1.3
+    Chunks initialized : 100/100
```
I think this section should be in a [code block](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code).

MSanKeys963 replied:
This section should be in a [code block](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code).

MSanKeys963 commented on docs/tutorial.rst:
```diff
@@ -204,6 +204,47 @@ here is an array using Zstandard compression, level 1::
+    Chunks initialized : 100/100
+
+
+Here is also an array using GZip compression, level 2::
+
+    >>> from numcodecs import GZip
+    >>> z = zarr.array(np.arange(100000000, dtype='i4').reshape(10000, 10000),
+    ...                chunks=(1000, 1000), compressor=GZip(level=2))
+    >>> z.compressor
+    GZip(level=2)
```
This too.

MSanKeys963 commented on docs/tutorial.rst:
```diff
@@ -204,6 +204,47 @@ here is an array using Zstandard compression, level 1::
+
+Getting the diagnostics on the compression ratio :
+
+    >>> z.info
+    Type               : zarr.core.Array
+    Data type          : int32
+    Shape              : (10000, 10000)
+    Chunk shape        : (1000, 1000)
+    Order              : C
+    Read-only          : False
+    Compressor         : GZip(level=2)
+    Store type         : zarr.storage.KVStore
+    No. bytes          : 400000000 (381.5M)
+    No. bytes stored   : 138601771 (132.2M)
+    Storage ratio      : 2.9
+    Chunks initialized : 100/100
```
This as well.

MSanKeys963 commented on docs/tutorial.rst:
```diff
@@ -204,6 +204,47 @@ here is an array using Zstandard compression, level 1::
+
+Here is also an array using GZip compression, level 2::
+
+    >>> from numcodecs import GZip
+    >>> z = zarr.array(np.arange(100000000, dtype='i4').reshape(10000, 10000),
+    ...                chunks=(1000, 1000), compressor=GZip(level=2))
+    >>> z.compressor
+    GZip(level=2)
```
This too.

Mubarraqqq replied:
Sorry that was my mistake @MSanKeys963.
I would work on that right away!

Mubarraqqq replied:
Hello @MSanKeys963 . I have resolved the issue

MSanKeys963 commented:
Hi @Mubarraqqq. I've added some minor changes here: https://github.com/Mubarraqqq/zarr-python/pull/1
Please have a look.

Mubarraqqq commented:
> Hi @Mubarraqqq. I've added some minor changes here: [Mubarraqqq#1](https://github.com/Mubarraqqq/zarr-python/pull/1) Please have a look.

Okay, I will have a look. 

> Hi @Mubarraqqq. I've added some minor changes here: [Mubarraqqq#1](https://github.com/Mubarraqqq/zarr-python/pull/1) Please have a look.

I've checked it out and merged it @MSanKeys963

jhamman commented:
This has unfortunately gone stale. Will it be revived or should we close this?

jhamman commented:
Closing as stale. Feel free to reopen if you want to revive the effort.
